desktop/apps/consignments2/client/actions.js

```diff
@@ -246,6 +296,35 @@ export function showNotConsigningMessage () {
   }
 }
 
+export function showResetPasswordSuccessMessage () {
+  return {
+    type: SHOW_RESET_PASSWORD_SUCESS_MESSAGE
+  }
+}
+
+export function signUp (values) {
+  return async (dispatch, getState) => {
+    try {
+      const options = {
+        email: values.email,
+        name: values.name,
+        password: values.password,
+        _csrf: sd.CSRF_TOKEN
+      }
+      await request
+              .post(sd.AP.signupPagePath)
+              .set('Content-Type', 'application/json')
+              .set('Accept', 'application/json')
+              .set('X-Requested-With', 'XMLHttpRequest')
+              .send(options)
+
+      await dispatch(logIn(values))
```

Comment:
Is this valid? Dispatch shouldn't return a promise unless you've implemented some kind of underlying lib, making `await` here invalid.